src/main/java/org/kairosdb/core/health/DatastoreQueryHealthCheck.java

```diff
@@ -28,10 +58,12 @@ public String getName() {
 
     @Override
     protected Result check() throws Exception {
-        // TODO(ville): Make the query configurable
         try (final DatastoreQuery query = datastore.createQuery(
-                new QueryMetric(System.currentTimeMillis() - (10 * 60 * 1000),
-                        0, "jvm/threads/thread_count"))) {
+                new QueryMetric(
+                        System.currentTimeMillis() - (10 * 60 * 1000),
```

Comment:
Should we make this time (10 min) configurable as well?

Reply:
Actually, probably not necessary now that I think about it. I think the choice of metric matters more than lookback for a healthcheck.

Reply:
That was my thinking as well, although it's easy to make it configurable.